business_requirement_deliverable_project/views/business_view.xml

```diff
@@ -41,7 +41,7 @@
                             type="object"
                             class="oe_highlight"
                             groups="business_requirement.group_business_requirement_manager"
-                            attrs="{'invisible': ['|', ('project_id', '=', False), ('state', '!=', 'approved')]}"/>
+                            attrs="{'invisible': ['|', ('project_id', '=', False), ('state', '!=', 'stakeholder_approval')]}"/>
```

Comment:
@elicoidal based on https://github.com/OCA/business-requirement/blob/8.0/business_requirement/models/business.py#L218, @Omarius1981 fix is correct.